src/app/state/transaction/types.ts

```diff
@@ -56,6 +56,15 @@ export interface TransactionState {
   success: boolean
   error?: ErrorPayload
   preview?: TransactionPreview
+  // TODO: Should pendingTransactions be persisted?
+  /**
+   * Keeps track of pending transactions.
+   *
+   * @typedef {Object} pendingTransactions
+   * @property {Object<string, Transaction[]>} [pendingTransactions] - A dictionary where the keys are addresses
+   *    and the value is array of pending transactions.
+   */
+  pendingTransactions: Record<string, Transaction[]>
```

Comment:
Should probably be in AccountState

Reply:
Will move, but not sure if it breaks migration or anything else.